### ISSUE TITLE
Sharing Empty State Illustration

### ISSUE THREAD

**Issue by glenn-sorrentino** [onionshare/onionshare-ios#22]
Current:
iOS empty state illustration is inconsistent with the Android app.

Expected:
Update with the assets found here: https://keybase.pub/glennsorrentino/onionshare/ios/illustrations/

![sharing-empty-state](https://user-images.githubusercontent.com/28545431/170809650-b485baad-10c7-4dd0-8c4b-46b5aa93c458.png)

**Comment by tladesignz** [comment]
Yeah, I didn't change this when it arrived, because I already had used images from the desktop app.

In fact, the Android app is inconsistent with all others now.

Isn't the desktop app the source of truth here?
